storage/gcloud/aio/storage/storage.py

```diff
@@ -59,31 +79,34 @@ def get_bucket(self, bucket_name: str) -> Bucket:
         }
 
         if not self.session:
-            self.session = aiohttp.ClientSession(conn_timeout=10,
-                                                 read_timeout=10)
-        session = session or self.session
-        resp = await session.delete(url, headers=headers, params=params or {},
-                                    timeout=timeout)
-        resp.raise_for_status()
```

Comment:
Why no more `raise_for_status()`?

Reply:
@jonathan-johnston looks like `raise_for_status()` is now called within the `session.{get,post,etc}()` methods. I could go either way on whether its better to have this be implicit -- I do kind of like the code reduction and guarantee of handling errors.

Reply:
Anyone has more thoughts on this? We can go either way on this...

Reply:
If no one else has comments: I think how you've refactored this is lovely :+1: